### DISCUSSION TITLE
(Projects beta) Create a project in a repo that automatically takes in issues that are given a particular label

### DISCUSSION THREAD

**Discussion by JaKXz** [community/community#5592] [Projects and Issues]
Hi, I am wondering if it is possible with the new GitHub Projects beta to create a project that automatically pulls in new issues that are given certain labels. I'm picturing an experience like Notion's "filtered views" where you can see cards that match certain criteria.

I would like to make a project board for "tech debt" [and keep the issues in our main project board as well], while being able to see all the progress/status of tech debt issues.

**Comment by Totktonada** [comment]
I have a similar (but more general) feature request: in https://github.com/github/feedback/discussions/5528 I asked to create a table, which includes all issues of an organization. Having it one can create any necessary view: say, issues with particular label (backlog) or issues without labels/milestones (to triage them).

**Reply by Anson115413** [reply]
Have ever open the files … it will be on public for a a month it’s been done before. The proposed of the project is not about to make a project base from the project .

**Reply by Anson115413** [reply]
> Hi, I am wondering if it is possible with the new GitHub Projects beta to create a project that automatically pulls in new issues that are given certain labels. I'm picturing an experience like Notion's "filtered views" where you can see cards that match certain criteria.
> 
> I would like to make a project board for "tech debt" [and keep the issues in our main project board as well], while being able to see all the progress/status of tech debt issues.

Have you open the files

**Comment by olafurpg** [comment]
I was hoping to re-create Jira "queues" using the new project beta but this limitation currently makes it unpractical because you have to manually add issues to the project board. Ideally, this functionality shouldn't be limited to labels. It should be possible to configure the board to automatically add issues that match a given query like `label:example created:<YYYY-MM-DD`.
